src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const CENTURY_DIVIDER = 100;
+  const filteredPeople = people.filter(person =>
+    (century !== undefined
```

Comment:
The filter operation for 'filteredPeople' might result in an empty array if no conditions are met. It's a good idea to add a check if the 'filteredPeople' array is not empty before performing calculations on it.